### DISCUSSION TITLE
Is there any way to add a profile to a list without following it first?

### DISCUSSION THREAD

**Discussion by daks** [mastodon/mastodon#28678] [Q&A] [answered]
Hi,
the question is in the title :) Is there any way to add a user profile to a list, without first needing to following the profile?

**Comment by ThisIsMissEm** [comment, marked as answer]
See: https://github.com/mastodon/mastodon/issues/6982

**Comment by daks** [comment]
The answer is no. But you can suscribe to an account and hide these posts from home, see https://github.com/mastodon/mastodon/issues/6982#issuecomment-1731998286
